<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComplexUIWindClass</class>
 <widget class="QMainWindow" name="ComplexUIWindClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ComplexUIWind</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ComplexUIWind.qrc">
    <normaloff>:/ComplexUIWind/Resources/MSN.ICO</normaloff>:/ComplexUIWind/Resources/MSN.ICO</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="txtEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>538</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionClear"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>格式</string>
    </property>
    <addaction name="actionBold"/>
    <addaction name="actionItalic"/>
    <addaction name="actionUnder"/>
    <addaction name="separator"/>
    <addaction name="actionShowTool"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionClose"/>
   <addaction name="separator"/>
   <addaction name="actionCut"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="separator"/>
   <addaction name="actionClear"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionBold"/>
   <addaction name="actionItalic"/>
   <addaction name="actionUnder"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QToolBar" name="tbCreate">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="ComplexUIWind.qrc">
     <normaloff>:/ComplexUIWind/Resources/FOLDER.ICO</normaloff>:/ComplexUIWind/Resources/FOLDER.ICO</iconset>
   </property>
   <property name="text">
    <string>新建...</string>
   </property>
   <property name="toolTip">
    <string>新建文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="ComplexUIWind.qrc">
     <normaloff>:/ComplexUIWind/Resources/MYSL.ICO</normaloff>:/ComplexUIWind/Resources/MYSL.ICO</iconset>
   </property>
   <property name="text">
    <string>打开...</string>
   </property>
   <property name="toolTip">
    <string>打开文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="ComplexUIWind.qrc">
     <normaloff>:/ComplexUIWind/Resources/OOFL.ICO</normaloff>:/ComplexUIWind/Resources/OOFL.ICO</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出程序</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="ComplexUIWind.qrc">
     <normaloff>:/ComplexUIWind/Resources/CNFNOT.ICO</normaloff>:/ComplexUIWind/Resources/CNFNOT.ICO</iconset>
   </property>
   <property name="text">
    <string>剪切</string>
   </property>
   <property name="toolTip">
    <string>剪切文本</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="ComplexUIWind.qrc">
     <normaloff>:/ComplexUIWind/Resources/DISTLSTL.ICO</normaloff>:/ComplexUIWind/Resources/DISTLSTL.ICO</iconset>
   </property>
   <property name="text">
    <string>复制</string>
   </property>
   <property name="toolTip">
    <string>复制文本</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="ComplexUIWind.qrc">
     <normaloff>:/ComplexUIWind/Resources/EXITEML.ICO</normaloff>:/ComplexUIWind/Resources/EXITEML.ICO</iconset>
   </property>
   <property name="text">
    <string>粘贴</string>
   </property>
   <property name="toolTip">
    <string>粘贴文本</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="ComplexUIWind.qrc">
     <normaloff>:/ComplexUIWind/Resources/POSTITL.ICO</normaloff>:/ComplexUIWind/Resources/POSTITL.ICO</iconset>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
   <property name="toolTip">
    <string>清空文本</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionBold">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="ComplexUIWind.qrc">
     <normaloff>:/ComplexUIWind/Resources/APPTL.ICO</normaloff>:/ComplexUIWind/Resources/APPTL.ICO</iconset>
   </property>
   <property name="text">
    <string>粗体</string>
   </property>
   <property name="toolTip">
    <string>粗体</string>
   </property>
  </action>
  <action name="actionItalic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="ComplexUIWind.qrc">
     <normaloff>:/ComplexUIWind/Resources/CONTACTL.ICO</normaloff>:/ComplexUIWind/Resources/CONTACTL.ICO</iconset>
   </property>
   <property name="text">
    <string>斜体</string>
   </property>
   <property name="toolTip">
    <string>斜体</string>
   </property>
  </action>
  <action name="actionUnder">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="ComplexUIWind.qrc">
     <normaloff>:/ComplexUIWind/Resources/EXITEML.ICO</normaloff>:/ComplexUIWind/Resources/EXITEML.ICO</iconset>
   </property>
   <property name="text">
    <string>下划线</string>
   </property>
   <property name="toolTip">
    <string>下划线</string>
   </property>
  </action>
  <action name="actionShowTool">
   <property name="icon">
    <iconset resource="ComplexUIWind.qrc">
     <normaloff>:/ComplexUIWind/Resources/MSN.ICO</normaloff>:/ComplexUIWind/Resources/MSN.ICO</iconset>
   </property>
   <property name="text">
    <string>显示工具栏文字标签</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ComplexUIWind.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionCut</sender>
   <signal>triggered()</signal>
   <receiver>txtEdit</receiver>
   <slot>cut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>227</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCopy</sender>
   <signal>triggered()</signal>
   <receiver>txtEdit</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>227</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPaste</sender>
   <signal>triggered()</signal>
   <receiver>txtEdit</receiver>
   <slot>paste()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>227</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionClose</sender>
   <signal>triggered()</signal>
   <receiver>ComplexUIWindClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionClear</sender>
   <signal>triggered()</signal>
   <receiver>txtEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>227</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtEdit</sender>
   <signal>copyAvailable(bool)</signal>
   <receiver>ComplexUIWindClass</receiver>
   <slot>on_txtEdit_copyAvailable(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel">
     <x>389</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtEdit</sender>
   <signal>selectionChanged()</signal>
   <receiver>ComplexUIWindClass</receiver>
   <slot>on_txtEdit_selectionChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>231</y>
    </hint>
    <hint type="destinationlabel">
     <x>389</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_txtEdit_copyAvailable(bool)</slot>
  <slot>on_txtEdit_selectionChanged()</slot>
 </slots>
</ui>
